applications/NXxas.nxdl.xml

```diff
@@ -116,8 +116,18 @@
             </field>
         </group>
         <group type="NXdata">
-          <link name="energy" target="/NXentry/NXinstrument/NXmonochromator:monochromator/energy"></link>
-          <link name="absorbed_beam" target="/NXentry/NXinstrument/NXdetector:absorbed_beam/data"></link>
+          <link name="energy" target="/NXentry/NXinstrument/NXmonochromator:monochromator/energy"/>
```

Comment:
Per https://manual.nexusformat.org/design.html#index-15, swap name and NX_class: `monochromator:NXmonochromator`